es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -48,10 +48,18 @@ properties:
     type: keyword
   days_to_index:
     type: long
+  days_to_lost_to_followup:
+    type: long
   demographic:
     properties:
+      cause_of_death:
+        type: keyword
       created_datetime:
         type: keyword
+      days_to_birth:
+        type: long
+      days_to_death:
```

Comment:
We are creating several AB instances in the dictionary with this minirelease. There will be duplicate properties until we successfully migrate the existing fields. Please refer to the individual tickets in the release ticket for the reasoning behind the various additions.